lms/djangoapps/django_comment_client/utils.py

```diff
@@ -652,7 +659,7 @@ def add_courseware_context(content_list, course, user, id_map=None):
             content.update({"courseware_url": url, "courseware_title": title})
 
 
-def prepare_content(content, course_key, is_staff=False, course_is_cohorted=None):
+def prepare_content(content, course_key, is_staff=False, discussion_division_enabled=None):
```

Comment:
This method makes a recursive call, and during the recursive calls, discussion_division_enabled will be defined (to avoid having to recalculate it). However, the original caller of the method will not pass this value in.

I kept the logic as it was and don't want to get into changing it. I will add comments though in the argument documentation.